foreman/data_refinery_foreman/surveyor/geo.py

```diff
@@ -116,11 +115,11 @@ def set_platform_properties(
             # Example of it being a list: GSE113945
             channel1_temp = sample_metadata.get("label_protocol_ch1", "")
             if type(channel1_temp) != list:
```

Comment:
This looks like a bug. I imagine we were hoping to do something with channel1_protocol (otherwise, why are we doing any of this?).

Reply:
looks like we could remove the whole `if...` since `channel1_temp` is also not used below

Reply:
If I had to guess I'd guess it had to with https://github.com/AlexsLemonade/refinebio/issues/87 or https://github.com/AlexsLemonade/refinebio/issues/89